mlrun/model_monitoring/db/stores/base/store.py

```diff
@@ -143,3 +143,41 @@ def get_endpoint_real_time_metrics(
         """
 
         pass
+
+    @abstractmethod
+    def write_application_result(self, event: dict[str, typing.Any]):
+        """
+        Write a new application result event in the target table.
+
+        :param event: An event dictionary that represents the application result, should be corresponded to the
+                      schema defined in the :py:class:`~mlrun.common.schemas.model_monitoring.constants.WriterEvent`
+                      object.
+        """
+        pass
+
+    @abstractmethod
+    def get_last_analyzed(self, endpoint_id: str, application_name: str) -> int:
```

Comment:
Future suggestion - add get_last_analyzed per endpoint only to improve our performance in the controller, and to access only one time per endpoint.

Reply:
cool, in general the new SQL structure already support it (by passing an empty application name) but we need to do a similar implementation for the KV structure.